src/main/main.cpp

```diff
@@ -81,9 +83,22 @@ auto main(int argc, char *argv[]) -> int {
   parser.SetStartSymbol(grammar_parser.GetStartSymbol());
   parser.SetParsingTable(parsing_table);
 
+  auto err = parsing_table.GetErrors();
+  if (!err.empty()) {
+    for (auto &e : err) {
+      std::cout << e << "\n";
+      return 0;
```

Comment:
If there are errors we are already returning 0. So why does Parser class store a copy of parsing_table_errors?